From 2a8e3e3e06731db0a2ff33977b8919170beb0679 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Gunter=20K=C3=B6nigsmann?= <gunter@peterpall.de>
Date: Thu, 5 Oct 2017 19:37:02 +0200
Subject: [PATCH] Reverted the bugfix that allows to use :lisp as the first
 command of a worksheet as it seems to introduce a race condition on
 evaluating the whole worksheet.

---
 data/wxmathml.lisp.in | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/data/wxmathml.lisp.in b/data/wxmathml.lisp.in
index a93fe381..21f7b656 100644
--- a/data/wxmathml.lisp.in
+++ b/data/wxmathml.lisp.in
@@ -1918,8 +1918,3 @@
        (simplify (mfunction-call $printf t '"~a;" $i)))
      (simplify (mfunction-call $printf t '"</slide></mth>")))
     ))
-
-;; Work around a bug in :lisp
-;; (If the first is a :lisp command it doesn't create an input prompt and thus doesn't
-;; cause wxMaxima to continue evaluation).
-(setq *mread-prompt* "<PROMPT-P/>(%i1) <PROMPT-S/>")
-- 
2.14.1

